tests/test_integrations_locally.py

```diff
@@ -9,7 +9,7 @@
 
 class PySonarNotebooks(unittest.TestCase):
 
-    def modelTrainingDemoNotebook(self):
+    def model_Training_DemoNotebook(self):
```

Comment:
why not `model_training_demo_notebook`?

Reply:
I didn't check. I thought it was a file name. so i left it in caps.